_faq/supported-devices.html

```diff
@@ -0,0 +1,16 @@
+---
+categories:
+- player
+type: info
+title: "Supported devices"
+---
+<section>
+    <p>NewPipe supports phones, tablets, and TVs.  NewPipe is also compatible with Android TV.</p>
+    <p>On some devices, it may be beneficial to change the appearance manually.  This can be done under:</p.
+    <ui>
+    <li>Settings</li>
+    <li>Appearance</li>
+    <li>List view mode</li>
+    </ui>
+    <p>You can toggle between <tt>auto</tt>, <tt>list</tt> and <tt>grid</tt>.</p>
```

Comment:
```suggestion
    <p>On some devices, it may be beneficial to change the appearance manually. This can be done by changing Settings &gt; Appearance &gt; List view model, which can be toggled between <tt>auto</tt>, <tt>list</tt> and <tt>grid</tt>.</p>
```